pkg/types/openstack/machinepool.go

```diff
@@ -11,6 +11,14 @@ type MachinePool struct {
 	// The instances use ephemeral disks if not set.
 	// +optional
 	RootVolume *RootVolume `json:"rootVolume,omitempty"`
+
+	// AdditionalNetworkIDs contains IDs of additional networks for machines
+	// +optional
```

Comment:
include the details of the validations.. also the note mentioned at https://github.com/openshift/installer/pull/3291/files#diff-e32975a988b75f5459f56a7c4e5affe7R164